contextualize_eval.cpp

```diff
@@ -0,0 +1,88 @@
+#include "contextualize_eval.hpp"
+#include "ast.hpp"
+#include "eval.hpp"
+#include "backtrace.hpp"
+#include "to_string.hpp"
+#include "parser.hpp"
+
+namespace Sass {
+
+  Contextualize_Eval::Contextualize_Eval(Context& ctx, Eval* eval, Env* env, Backtrace* bt)
+  : Contextualize(ctx, env, bt), eval(eval)
+  { }
+
+  Contextualize_Eval::~Contextualize_Eval() { }
+
+  Selector* Contextualize_Eval::fallback_impl(AST_Node* n)
+  { return Contextualize::fallback_impl(n); }
```

Comment:
I think i see what you're saying.  I think I was thinking of it as that this class is a sort of decorator or just extension on the original contextualize class.  I started down the path you are suggesting and went with inheritance in this case because I felt that it felt more natural.  I can try it the other way again